tests/unit/data_utils/test_ml_feature_loader_and_pipeline.py

```
@@ -91,4 +91,5 @@ def test_prepare_ml_data_for_training_with_cleaning_uses_cache_when_available(mo
     mp._cleaned_data_cache.get_cache_age_hours = lambda *a, **k: 1.0
     mp._cleaned_data_cache.load_cleaned_data = lambda *a, **k: fake_cached
     res = mp.prepare_ml_data_for_training_with_cleaning()
-    assert res == fake_cached, "Cached result not returned when cache present"
+    if res != fake_cached:
+        raise AssertionError("Cached result not returned when cache present")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)